<launch>
<arg name="map_file" default="$(find arobota2)/map/mymap5.yaml"/>
<param name="/use_sim_time" value="false"/>

<!--global map-->
  <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(find arobota2)/rviz/navigation_multi_new4.rviz"/>
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/>
      <param name="frame_id" value="/map" />

    <include file="$(find arobota2)/launch/navigation_gazebo_multi_1.launch"/>
    <include file="$(find arobota2)/launch/navigation_gazebo_multi_2.launch"/>
    <include file="$(find arobota2)/launch/navigation_gazebo_multi_3.launch"/>
    <include file="$(find arobota2)/launch/multi_robot_des_real.launch"/>

  <!-- Nodes -->
  <group ns="robot1">
    <node pkg="arobota2" type="omni_odom_tf_multi_1.py" name="Omni_tf" output="screen"/>
      <!-- <param name ="base_frame_id" value= "robot1_tf/base_footprint"/>
      <param name ="odom_frame_id" value= "robot1_tf/odom"/> -->
    <node pkg="arobota2" type="omni_cmdveltf.py" name="cmd_vel_tf" output="screen"/>

    <!-- Robot state -->
    <!-- <include file="$(find arobota2)/launch/robot_remote.launch"/> -->

    <!-- Map server -->
    <!-- <arg name="map_file" default="$(find arobota2)/map/mymap5.yaml"/>
    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/> -->

    <!-- AMCL -->
    <!-- <include file="$(find arobota2)/launch/amcl_omni.launch"/> -->

    <!-- move_base -->
    <!-- <include file="$(find arobota2)/launch/movebase.launch"/> -->

    <!-- AMCL + move_base -->
    <!-- <include file="$(find arobota2)/launch/navigation_gazebo_multi_1.launch"/> -->

    <!-- rviz -->
  </group>

  <group ns="robot2">
    <node pkg="arobota2" type="omni_odom_tf_multi_2.py" name="Omni_tf" output="screen"/>
      <!-- <param name ="base_frame_id" value= "robot2_tf/base_footprint"/>
      <param name ="odom_frame_id" value= "robot2_tf/odom"/> -->
    <node pkg="arobota2" type="omni_cmdveltf.py" name="cmd_vel_tf" output="screen"/>

    <!-- Robot state -->
    <!-- <include file="$(find arobota2)/launch/robot_remote.launch"/> -->

    <!-- Map server -->
    <!-- <arg name="map_file" default="$(find arobota2)/map/mymap5.yaml"/>
    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/> -->

    <!-- AMCL -->
    <!-- <include file="$(find arobota2)/launch/amcl_omni.launch"/> -->

    <!-- move_base -->
    <!-- <include file="$(find arobota2)/launch/movebase.launch"/> -->

    <!-- AMCL + move_base -->
    <!-- <include file="$(find arobota2)/launch/navigation_gazebo_multi_2.launch"/> -->
    
    <!-- rviz -->
  </group>

  <group ns="robot3">
    <node pkg="arobota2" type="omni_odom_tf_multi_3.py" name="Omni_tf" output="screen"/>
      <!-- <param name ="base_frame_id" value= "robot3_tf/base_footprint"/>
      <param name ="odom_frame_id" value= "robot3_tf/odom"/> -->
    <node pkg="arobota2" type="omni_cmdveltf.py" name="cmd_vel_tf" output="screen"/>

    <!-- Robot state -->
    <!-- <include file="$(find arobota2)/launch/robot_remote.launch"/> -->

    <!-- Map server -->
    <!-- <arg name="map_file" default="$(find arobota2)/map/mymap5.yaml"/>
    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/> -->

    <!-- AMCL -->
    <!-- <include file="$(find arobota2)/launch/amcl_omni.launch"/> -->

    <!-- move_base -->
    <!-- <include file="$(find arobota2)/launch/movebase.launch"/> -->

    <!-- AMCL + move_base -->
    <!-- <include file="$(find arobota2)/launch/navigation_gazebo_multi_3.launch"/> -->

    <!-- rviz -->
  </group>

</launch>
